crates/matrix-sdk/src/encryption/mod.rs

```diff
@@ -404,10 +404,13 @@ impl Client {
     fn get_dm_room(&self, user_id: &UserId) -> Option<room::Joined> {
         let rooms = self.joined_rooms();
         let room_pairs: Vec<_> =
-            rooms.iter().map(|r| (r.room_id().to_owned(), r.direct_target())).collect();
+            rooms.iter().map(|r| (r.room_id().to_owned(), r.direct_targets())).collect();
         trace!(rooms = ?room_pairs, "Finding direct room");
```

Comment:
Creating this heavy room_pairs just for the trace? Even though it isn't even used to find the room? Let's remove this.

Reply:
It was already there, and i was wondering the same. Will remove it :)